.ts_pre_commit_config.yaml

```diff
@@ -3,4 +3,4 @@ check-xml: true
 black: true
 flake8: true
 isort: true
-mypy: false
+mypy: false
```

Comment:
make sure you leave an empty line here at the end.  There should be no change to this file with respect to develop.